### PR TITLE
Add PVTA and MBTA

### PR DESCRIPTION
**Pull request by isabelle-dr**
Add PVTA and MBTA in the list of fares v2 first adopters after now that https://github.com/google/transit/pull/385 has passed

### CODE REVIEW

File: docs/extensions/fares-v2.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can stay up to date and join the discussions around Fares v2 by joining our
...
 
-- Producers: <a href="https://www.interline.io/" target="_blank">Interline</a>, <a href="https://www.mta.maryland.gov/developer-resources" target="_blank">Maryland Department of Transportation</a>, <a href="https://dot.ca.gov/cal-itp/cal-itp-gtfs" target="_blank">Cal-ITP</a>, <a href="https://trilliumtransit.com/" target="_blank">Trillium Solutions</a>, <a href="https://www.itoworld.com/" target="_blank">ITO World</a>
+- Producers: <a href="https://www.interline.io/" target="_blank">Interline</a>, <a href="https://www.mta.maryland.gov/developer-resources" target="_blank">Maryland Department of Transportation</a>, <a href="https://dot.ca.gov/cal-itp/cal-itp-gtfs" target="_blank">Cal-ITP</a>, <a href="https://trilliumtransit.com/" target="_blank">Trillium Solutions</a>, <a href="https://www.itoworld.com/" target="_blank">ITO World</a>, <a href="https://www.mbta.com/" target="_blank">MBTA</a>, <a href="https://www.pvta.com/index.php/" target="_blank">PVTA</a>
```
Comment (tzujenchanmbd) [inline docs/extensions/fares-v2.md]: Suggest changing PVTA's url to "http://www.pvta.com/" (remove "index.php").
The index.php one works a bit weird on my end, while just pvta.com works well.

**Review by tzujenchanmbd — COMMENTED** [review]
Suggest changing url for PVTA

File: docs/extensions/fares-v2.md (hunk 1 above)
Comment (isabelle-dr) [reply docs/extensions/fares-v2.md]: Done in the last commit, can you have another look?

**Review by tzujenchanmbd — APPROVED** [review]
lgtm!

File: docs/extensions/fares-v2.md (hunk 1 above)
Comment (tzujenchanmbd) [reply docs/extensions/fares-v2.md]: approved!

**Pull request merged**